package.json

```diff
@@ -31,7 +31,7 @@
     "@manuscripts/assets": "^0.6.2",
     "@manuscripts/comment-editor": "^1.0.3",
     "@manuscripts/json-schema": "^2.1.0",
-    "@manuscripts/library": "^1.1.2",
+    "@manuscripts/library": "^1.1.2-LEAN-2501-test",
     "@manuscripts/style-guide": "^1.1.14-LEAN-2501-test",
```

Comment:
can you please update this to your version

Reply:
which version do you mean?
library is currently on 1.1.2-LEAN-2501-test

Reply:
there is a mismatch with the `style-guide` check acjenkins log. did you cut the branch from the master?